client/reader/search-stream/index.jsx

```diff
@@ -16,14 +16,17 @@ import EmptyContent from './empty';
 import BlankContent from './blank';
 import HeaderBack from 'reader/header-back';
 import SearchInput from 'components/search';
-import SearchCard from 'blocks/reader-search-card';
 import SiteStore from 'lib/reader-site-store';
 import FeedStore from 'lib/feed-store';
 import { recordTrackForPost } from 'reader/stats';
 import sampleSize from 'lodash/sampleSize';
 import i18nUtils from 'lib/i18n-utils';
 import { staffSuggestions, popularSuggestions } from './suggestions';
 import { abtest } from 'lib/abtest';
+import SearchCard from 'blocks/reader-search-card';
+import ReaderPostCard from 'blocks/reader-post-card';
+import config from 'config';
+import { localize } from 'i18n-calypso';
```

Comment:
Tiniest of nitpicks - with 'i18n-calypso' being an external package it should probably live in the externals above (even though it's Calypso specific).  
Also in the same vein, but not strictly part of this PR - there's reference to lodash here in the internals (`sampleSize`) which could be merged in to the reference at the top :)